src/index.ts

```diff
@@ -237,7 +238,6 @@ const plugin: PluginImpl<{
     },
     generateBundle() {
       compiledTemplates.clear();
-      virtualFiles.clear();
```

Comment:
I just realised that you could keep this line with an " if (!this.meta.watchMode) " in front, but it only exists since [v2.14.0](https://github.com/rollup/rollup/releases/tag/v2.14.0) , so ...

Reply:
@AurelienStebe I'm not too concerned since my guess is if you aren't in watch mode this process probably isn't long-lived anyways.

There is probably a way we could remove virtual files for templates that no longer exist, but I don't think it's a big deal for now.

Reply:
My thoughts exactly.